app/views/layouts/angular-bootstrap/_auth_credentials_angular_bootstrap.html.haml

```diff
@@ -32,7 +33,7 @@
       %input.form-control{"type"         => "password",
                           "id"           => "textInput-markup",
                           "ng-required"  => "#{ng_reqd_password}",
-                          "placeholder"  => "#{placeholder_if_present("#{ng_model}.#{prefix}_password")}",
+                          "placeholder"  => "",
```

Comment:
well, at least used to

here I don't get how the stuff was supposed to work prior to this PR :-(